sys/auto_init/auto_init.c

```diff
@@ -112,6 +112,10 @@ void auto_init(void)
 #ifdef MODULE_SCHEDSTATISTICS
     init_schedstatistics();
 #endif
+#ifdef MODULE_EVENT_THREAD
+    extern void auto_init_event_thread(void);
+    auto_init_event_thread();
```

Comment:
This is by the way much better then spawning by demand at the first posting of an event, as the first event is likely to be posted from IRQ context. I'd much rather have more overhead in the boot up than in IRQ context.